azure/packages/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -225,7 +204,7 @@ describe("AzureClient", () => {
 
         const { container: containerGet } = await client.getContainer(containerId, schema);
         const map1Get = containerGet.initialObjects.map1 as SharedMap;
-        const valueGet: string | undefined = await mapWait(map1Get, "new-key");
+        const valueGet: string | undefined = await map1Get.get("new-key");
```

Comment:
I think the wait may be necessary here to ensure reliability, as there's no mechanism in place to guarantee that the new container will see the set prior to attempting the get.

Reply:
Also I see a number of already-existing instances of `await` being used on calls to `map.get()` which isn't necessary as it's a synchronous API.

Reply:
Restore the mapWait here